config/webpack/environment.js

```diff
@@ -1,5 +1,15 @@
 const { environment } = require('@rails/webpacker')
+const webpack = require('webpack')
 // const vue =  require('./loaders/vue')
 
+environment.plugins.append(
+  'Provide',
+  new webpack.ProvidePlugin({
+    $: 'jquery',
+    jQuery: 'jquery',
+    'window.jQuery': 'jquery',
+  })
+)
+
```

Comment:
Hi @cacquani, how come we need to expose these? It looks like your JS is all inside webpack.

Reply:
Without this config setting, I can't access the actual `$` or `jQuery` vars correctly in the `nest.js`, even if it's all inside webpack.

Anyway, here it suggests to do it like this to have jQuery available in the `app/javascript/packs/application.js`, and since it works I assumed it was the right way to do it:
https://www.botreetechnologies.com/blog/introducing-jquery-in-rails-6-using-webpacker

Reply:
Hi @cacquani, have you tried importing jQuery in the way we import it in other parts of our JavaScript (`import $ from 'jquery'`)?

The `provide` functionality in WebPack autoloads jQuery, etc., but this is less explicit and loads it into contexts even when we don't need it. The jury's out on whether this is a "better" approach than explicitly loading dependencies as we have done elsewhere, but it's probably better for us to use a consistent approach within F&E.